.github/workflows/ci.yml

```diff
@@ -29,12 +29,17 @@ jobs:
 
       - name: Build Project
         run: yarn build
-
-      # commenting for now to make CI green , will try this 
-      # - name: run Lighthouse CI
-      #   run: |
-      #     yarn global add @lhci/cli@0.8.x
-      #     lhci autorun --collect.staticDistDir
-
+      
```

Comment:
in a subsequent PR let's integrate support for this

Reply:
Sure